airbyte-integrations/connectors/source-grafana/source_grafana/grafana.yaml

```diff
@@ -0,0 +1,40 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "{{ config['url'] }}/api"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config['api_key'] }}"     
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  datasources_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "datasources"
+      path: "/datasources"
+  reports_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "reports"
+      path: "/reports" 
```

Comment:
@bala-ceg as described in low-code CDK async endpoints aren't implemented yet. I don't think implementing Grafana using low-code you can reach the goal of this connector. Also I don't see the /report endpoint inlatest version of the API.